sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -6,7 +6,6 @@
  * @brief #Azure::Core::Http::HttpTransport implementation via WinHTTP.
  * cspell:words HCERTIFICATECHAIN PCCERT CCERT HCERTCHAINENGINE HCERTSTORE
```

Comment:
cspell comments should not be in the doxygen comment sections. It was almost correct before (cspell directive was as a regular comment below, but doxygen section also had it).